cl_dll/view.cpp

```diff
@@ -90,6 +90,7 @@ cvar_t	*cl_bob;
 cvar_t	*cl_bobup;
 cvar_t	*cl_waterdist;
 cvar_t	*cl_chasedist;
+cvar_t* cl_wonbob;
```

Comment:
I suggest calling it `bobwon` rather than `wonbob` so it shows up when you type `cl_bob` among the other variables. Even better, something like `bobtilt` so it's actually a descriptive name.

Reply:
Changes made in commit 296734c